test/cookbooks/osl-rt-test/recipes/osl_request_tracker.rb

```diff
@@ -0,0 +1,28 @@
+# Download mailx for testing the email queue later
+package %w(mailx jq)
+
+# Database
+osl_mysql_test 'rt' do
+  username node['osl-rt']['db']['username']
+  password node['osl-rt']['db']['password']
+end
+
+# Request Tracker
+include_recipe 'osl-rt'
+
+# Wait a bit so the RT instance could start back up
+chef_sleep 'Wait a bit' do
+  seconds 5
```

Comment:
Just do this to keep it cleaner.
```suggestion
chef_sleep '5' do
```